Tests/test_raxml_tool.py

```diff
@@ -24,7 +25,9 @@
 
 if not raxml_exe:
     raise MissingExternalDependencyError(
-        "Install RAxML (binary raxmlHPC) if you want to test the Bio.Phylo.Applications wrapper.")
+        "Install RAxML (binary raxmlHPC) if you want"
+        "to test the Bio.Phylo.Applications wrapper."
+    )
```

Comment:
Missing the space at the split point - opinion is divided over a trailing space vs a leading space. Doing the later makes it easy to spot missing spaces.

Reply:
Added a leading space, thanks for spotting it! Will be more careful